ci-templates/gitlab/k8s-deploy.yml

```
@@ -0,0 +1,97 @@
+# SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+# SPDX-License-Identifier: Apache-2.0
+
+variables:
+  GRAFANA_HELM_CHART: https://grafana.github.io/helm-charts/
+  PRIVATE_GPG_PATH: /secrets/private.gpg
+  TARGET:
+    value: "staging"
+    description: "Specify the target (staging/production)"
+  REVISION:
+    value: "main"
+    description: "Revision of the Github repository (capella-collab-manager)"
+
+
+.decrypt: &decrypt
+  - apt-get install -y jq
+  - cd ${TARGET}
+  - gpg --import ${PRIVATE_GPG_PATH}
+  - sops -d secret.values.yaml > ../plain.values.yaml
+  - sops -d secret.k8s.json > ../plain.k8s.json
+  - cd ..
+  # Remove _unencrypted suffix from plain.k8s.json
+  - |
+    python3 <<EOF
+    import json
+    import pathlib
+    import typing as t
+
+    def strip_suffix(data: t.Any):
+        if isinstance(data, dict):
+            return {key.replace('_unencrypted', ''): strip_suffix(value) for key, value in data.items()}
+        elif isinstance(data, list):
+            return [strip_suffix(element) for element in data]
+        else:
+            return data
+
+    path = pathlib.Path("plain.k8s.json")
+    data = json.loads(path.read_text())
+    path.write_text(json.dumps(strip_suffix(data), indent=4))
+    EOF
+
+.prepare: &prepare
+    - *decrypt
+    - git clone https://github.com/DSD-DBS/capella-collab-manager
+    - cd capella-collab-manager
+    - git checkout ${REVISION}
+    - sed -i "s|https://grafana.github.io/helm-charts/|${GRAFANA_HELM_CHART}|g" ./helm/Chart.yaml
```

```suggestion
    - sed -i "s#https://grafana.github.io/helm-charts/#${GRAFANA_HELM_CHART}#g" ./helm/Chart.yaml
```

I don't like the use of the pipe symbol here. It is already reserved in Linux can be confusing for the user.

---

Done